lib/eyeshade/wallet.rb

```diff
@@ -38,9 +38,9 @@ def initialize(wallet_json:, channel_json:)
         @channel_balances[identifier] = Eyeshade::Balance.new(balance_json: json)
       end
 
-      if wallet_json["lastSettlement"]
+      if wallet_json["lastSettlement"].present?
         @last_settlement_balance = Eyeshade::Balance.new(balance_json: wallet_json["lastSettlement"].merge({'rates' => @rates}), apply_fee: false)
-        @last_settlement_date = Time.at(wallet_json["lastSettlement"]["timestamp"]/1000).to_datetime
+        @last_settlement_date = Time.at(wallet_json["lastSettlement"]["timestamp"]).to_datetime
```

Comment:
I'm not sure why we dividing by 1000 previously

Reply:
nodejs timestamps are usually in milliseconds - that probably got exposed over the api